spec/lib/core_simulator_spec.rb

```diff
@@ -1311,6 +1311,12 @@ def create_sandbox_dirs(sub_dir_names)
 
       Resources.shared.alt_xcode_install_paths.each do |path|
         it "#{path}" do
+          # On jenkins machine we have macOS 10.12.6 installed
+          # Xcode >= 9.3 requires at least macOS 10.13
+          # so we skip these tests for now
+          if !ENV["JENKINS_HOME"].nil? && !path[/9\.[3-9]/].nil?
+            skip "Can't test against Xcode >= 9.3 on CI machine with macOS 10.12.6"
+          end
```

Comment:
I installed Xcode 9.3 on the macOS machine by mistake.  We should delete the Xcode 9.3 install.

Sometime this summer we will update the Jenkins machine.

Reply:
Xcode 9.3 has been removed from the CI machine.

We can remove this check.